scripts/generator/graphql_generator/csharp/SDK/CheckoutUtils.cs.erb

```diff
@@ -0,0 +1,22 @@
+namespace <%= namespace %>.SDK {
+    using System.Collections.Generic;
+    using <%= namespace %>.MiniJSON;
+    
+    public class CheckoutUtils {
```

Comment:
I only needed this method from `iOSNativeCheckout` so I pulled it into a utility class but if I end up needed more I'll probably end up abstract base classing `INativeCheckout` down the road.